Python/initconfig.c

```diff
@@ -28,17 +28,23 @@
 static const char usage_line[] =
 "usage: %ls [option] ... [-c cmd | -m mod | file | -] [arg] ...\n";
 
-/* Long usage message, split into parts < 512 bytes */
+/* Long help messages, split into parts < 512 bytes */
+/* Lines sorted by option name; keep in sync with usage_envvars* below */
 static const char usage_1[] = "\
-Options and arguments (and corresponding environment variables):\n\
+Options (and corresponding environment variables):\n\
 -b     : issue warnings about str(bytes_instance), str(bytearray_instance)\n\
          and comparing bytes/bytearray with str. (-bb: issue errors)\n\
 -B     : don't write .pyc files on import; also PYTHONDONTWRITEBYTECODE=x\n\
 -c cmd : program passed in as string (terminates option list)\n\
+--check-hash-based-pycs always|default|never :\n\
+         control how Python invalidates hash-based .pyc files\n\
 -d     : turn on parser debugging output (for experts only, only works on\n\
          debug builds); also PYTHONDEBUG=x\n\
 -E     : ignore PYTHON* environment variables (such as PYTHONPATH)\n\
 -h     : print this help message and exit (also --help)\n\
+--help-env      : print help about Python environment variables and exit\n\
+--help-xoptions : print help about implementation-specific -X options and exit\n\
+--help-all      : print complete help information and exit\n\
 ";
 static const char usage_2[] = "\
```

Comment:
Would it be a problem to fold `usage_2` and `usage_3` into `usage_1`?  I'm guessing they're artifacts of when older compilers had limits on string literal size.